### PR TITLE
2 exercises

### PR DESCRIPTION
**Pull request by nitisharora9**

### CODE REVIEW

File: age_compare.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+<p id="output"></p>
+
+<script>
+class User {
+  constructor(name, age) {
+    this.name = name;
+    this.age = age;
+  }
+
+  compare(user){
```
Comment (shubhamjuneja1) [inline age_compare.html]: compare is not helping us to determine what this function is doing, please rename.

File: age_compare.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    const user1Age = this.age;
```
Comment (shubhamjuneja1) [inline age_compare.html]: Why these are const?

File: age_compare.html [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    const user2Age = user.age;
+    const user1Name = this.name;
+    const user2Name = user.name;
+
+    if(user1Age > user2Age){
+        document.getElementById("output").innerHTML = user1Name + " is older than " + user2Name;
```
Comment (shubhamjuneja1) [inline age_compare.html]: this should be extracted in a different method.

File: welcome.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<body onload="message()">
+
+<script>
+
+function message() {
```
Comment (shubhamjuneja1) [inline welcome.html]: don't you think, this should be called on document onload?

**Review by shubhamjuneja1 — CHANGES_REQUESTED** [review]

File: age_compare.html (hunk 1 above)
Comment (nitisharora9) [reply age_compare.html]: Okay I will rename it

**Pull request closed without merging**